app.py

```diff
@@ -176,13 +176,7 @@ def session_stats_create(session_key, caller_identity):
 @validate_json
 @recover_identity
 def node_send_stats(caller_identity):
-    payload = request.get_json(force=True)
-    node_key = payload.get('node_key', '')
-    sessions = payload.get('sessions', [])
-
-    return_values = []
-
-    node = Node.query.get(node_key)
```

Comment:
Is query still performed? Plz doublecheck if still have validation if updated record exist